### PR TITLE
Math-Magician Add Components

### PR DESCRIPTION
**Pull request by RichardChileya**
### In this Section, I: 

- Add a new branch 
- Added components directory in the source folder to hold all App components
- Added styling to the App
- Checked for linter error
- Fixed Linter errors 
- Updated the Readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ A simple calculator project built using ReactJS. In this section I configured th
  - Creating a new branch for the setup
  - Adding Linter files
  - Adding React libraries 
+ - Add Componets (Calculator, Display & Rows to the App)
 
 ## Built With
```
Comment (TedLivist) [inline README.md:9]: - [x] Kindly remove these highlighted boilerplate files.

  wed

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @RichardChileya,

Good job so far!
✔️ Good documentation
✔️ Use of class components
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (RichardChileya) [reply README.md:9]: Thank you for your feedback. the issue has been resolved

**Review by brytebee — APPROVED** [review]
Hi, @RichardChileya👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

##

###  **Optional Suggestions:**
- N/A

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**